Gemfile

```diff
@@ -52,6 +52,7 @@ end
 group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem 'annotate', '~> 3.2'
+  gem 'brakeman', '~> 5.2', '>= 5.2.3'
```

Comment:
Wasn't this already merged? 🤔

Reply:
@LeandroSoria2 Looks like this branch needs to be rebased